src/components/Action/Action.vue

```diff
@@ -100,16 +124,41 @@ export default {
 	methods: {
 		toggleMenu() {
 			this.opened = !this.opened
+			// focus first on menu open after opening the menu
+			if (this.opened) {
+				this.$nextTick(function() {
```

Comment:
while it might be okay here, you should rather use arrow functions for this type of code because then you know `this` will always point to the context of the outer function. with `function` it could be something different.

Reply:
Vue automatically bind this to any function in nexttick :)
Otherwise it would not work :wink: 
Though arrow function looks nicer, let's change it!